app/controllers/attendances_controller.rb

```diff
@@ -1,4 +1,44 @@
 class AttendancesController < ApplicationController
   def index
   end
+
+  def new
+    @attendance = Attendance.new
+    @company = Company.enabled
+  end
+
+  def create
+    @employee = Employee.enabled
+    data = params.require(:attendance).permit(:private_num, :company_id)
+    data[:check_in] = Time.now.strftime("%H:%M:%S")
+    data[:attendance_date] = Time.now.strftime("%d-%m-%Y")
+    emp = @employee.find_by(private_num: data.fetch(:private_num), company_id: data.fetch(:company_id))
```

Comment:
Now that I think about it, is there any reason to use `@employee` inside this action? Why not just do something like this instead:

```suggestion
    emp = Employee.enabled.find_by(data.slice(:private_num, :company_id))
```